test/case/ietf_syslog/remote/test.py

```diff
@@ -146,9 +145,11 @@
             }
         })
 
-    with test.step("Verify logging from client to server"):
-        clientssh.runsh("logger -t test -m client -p security.notice Hej")
+    with test.step("Send logging message with facility 'security' and severity 'notice' on client"):
+        clientssh.runsh("logger -t test -m client -p security.notice TestMessage")
+
+    with test.step("Verify the message exist in /log/security on server"):
```

Comment:
Maybe "Verify reception of client log message, incl. sorting to /log/security on server"?